### DISCUSSION TITLE
Cross platform EOL LF/CRLF configuration

### DISCUSSION THREAD

**Discussion by graffhyrum** [biomejs/biome#4512] [Q&A]
I'm trying to migrate to biome in a few projects and my team uses windows and linux for development. We are trying to figure out how to make `formatter.lineEnding` align with the host OS. Has anyone else dealt with this and found a solution? Is there a best practice from the biome team for configuring this?

**Comment by dyc3** [comment]
Biome doesn't have configuration to do this. Could you elaborate more on your use case? What's wrong with just picking one line ending?

**Reply by graffhyrum** [reply]
The issue is that biome doesn't seem to know/care about my `gitattributes` or `core.autocrlf`, so we have to either templatize and generate our biome conf per system to match the host, or forego using the formatter entirely (since as far as I know there's no way to disable just one formatter rule)

**Comment by ematipico** [comment]
If you need to deal with line ending on multiple OS, and you use a VCS, you should rely on it mainly.

With git, here's how we do it in the Biome repository: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#example
